### PR TITLE
adding code of conduct #25

### PR DESCRIPTION
**Pull request by VoidVampire**
I have added a sufficient code of conduct file as per #25

### CODE REVIEW

**Comment by VoidVampire** [comment]
I have a request. If my PR is good, can you merge it under hacktoberfest-accepted label, otherwise it won't be counted as far as I know.

**Review by Spydr06 — CHANGES_REQUESTED** [review]
I've stated a few changes necessary to proceed

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+opensource@dagworks.io.
```
[the remark this answers is not in the document]
Comment (VoidVampire) [reply CODE_OF_CONDUCT.md]: I have resolved other changes. For this particular change, linking your twitter was the best link I could find which could be used in this stance.
My other suggestion is to remove those 3 lines entirely but for now, I have not done that. Please check the new commit.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -60,59 +60,12 @@ representative at an online or offline event.
...
 
-## Enforcement Guidelines
-
-Community leaders will follow these Community Impact Guidelines in determining
-the consequences for any action they deem in violation of this Code of Conduct:
-
-### 1. Correction
-
-**Community Impact**: Use of inappropriate language or other behavior deemed
-unprofessional or unwelcome in the community.
```
Comment (Spydr06) [inline CODE_OF_CONDUCT.md]: imho, the guidelines seem pretty important for this, just leave out the "banning/warning" stuff

**Review by Spydr06 — CHANGES_REQUESTED** [review]

**Review by Spydr06 — APPROVED** [review]
Looks good, I might expand on this in the future

**Pull request merged**